src/Lykke.Service.BlockchainWallets/Controllers/ConstantsController.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Threading.Tasks;
+using Lykke.Common.Api.Contract.Responses;
+using Lykke.Service.BlockchainWallets.Contract.Models;
+using Lykke.Service.BlockchainWallets.Core.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Lykke.Service.BlockchainWallets.Controllers
+{
+    [Route("api/constants")]
+    public class ConstantsController : Controller
```

Comment:
Add a description for all API methods in controllers.

Reply:
Fixed